### DISCUSSION TITLE
Data Storage

### DISCUSSION THREAD

**Discussion by bramalingam81** [moneymanagerex/moneymanagerex#6851] [Q&A] [answered]
Hello Everyone,

Could you pls details on whether the data/transactions is stored locally after the software is installed ? Does the data gets uploaded to any external locations ? 

Please let me know on this. Thanks.

**Comment by renato-mmex** [comment, marked as answer]
The data is only stored locally in the databases .mmb + .db3
Statistical data on the use of MMEX that is transmitted to the MMEX server is clearly described and can be deactivated if desired (Tools/Settings/Network/Usage Statistics)

**Reply by bramalingam81** [reply]
Thank you for clarifying on this query.
